flixel/input/gamepad/FlxGamepad.hx

```diff
@@ -261,10 +269,10 @@ class FlxGamepad implements IFlxDestroyable
 		manager = null;
 		
 		#if FLX_JOYSTICK_API
-		hat = FlxDestroyUtil.put(hat);
+		hat  = FlxDestroyUtil.put(hat);
```

Comment:
Fyi, I'm not a fan of this formatting for vertical alignment. That's just a lot of extra maintenance effort with little benefit.